scripts/build_openupgrade_docs

```diff
@@ -1,36 +1,25 @@
 #!/bin/sh
 
-DOC_BUILD_BASE_DIR=/var/tmp/openupgrade
-DOC_PARTS="6.0|bin/openupgrade/doc
-6.1|openerp/openupgrade/doc
-7.0|openerp/openupgrade/doc
-8.0|openerp/openupgrade/doc
-9.0|openerp/openupgrade/doc
-10.0|odoo/openupgrade/doc"
-OUTPUT_DIR=${OUTPUT_DIR:-$DOC_BUILD_BASE_DIR/doc}
-DOC_BUILD_DIR=$(mktemp -d)
-REPO_URL=https://github.com/OCA/OpenUpgrade.git
-OPENUPGRADELIB=https://github.com/OCA/openupgradelib.git
+DOC_BUILD_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../odoo/openupgrade/doc && pwd )"
```

Comment:
this combines badly with the shebang above. Better say `$(realpath $(dirname $0)/../odoo/openupgrade/doc)`, this should run on basically any shell and only needs coreutils installed. It's a weird system if that's not the case.

But I don't really see the point of keeping the build dir, what's your rationale here?